official/resnet/imagenet_test.py

```diff
@@ -214,14 +213,6 @@ def test_resnet_model_fn_train_mode_v1(self):
   def test_resnet_model_fn_train_mode_v2(self):
     self.resnet_model_fn_helper(tf.estimator.ModeKeys.TRAIN, version=2)
 
-  def test_resnet_model_fn_train_mode_multi_gpu_v1(self):
-    self.resnet_model_fn_helper(tf.estimator.ModeKeys.TRAIN, version=1,
-                                multi_gpu=True)
-
-  def test_resnet_model_fn_train_mode_multi_gpu_v2(self):
-    self.resnet_model_fn_helper(tf.estimator.ModeKeys.TRAIN, version=2,
-                                multi_gpu=True)
```

Comment:
The thing is model_fn used to be device aware because of replicate_model_fn, and now it isn't. That's why I can safely remove that test. And we don't currently have the infrastructure for me to set up a multi-gpu end-to-end test.